bugsnag-android-core/src/test/java/com/bugsnag/android/InternalReportDelegateTest.kt

```diff
@@ -59,9 +56,7 @@ internal class InternalReportDelegateTest {
         delegate.reportInternalBugsnagError(event)
 
         // app
-        assertEquals(2, event.app["foo"])
-        assertEquals(500L, event.app["durationInForeground"])
-        assertEquals(true, event.app["inForeground"])
```

Comment:
I dislike removing assertions! Can't this data be easily set in the dummy app that was generated?

Reply:
I've added this assertion back in.